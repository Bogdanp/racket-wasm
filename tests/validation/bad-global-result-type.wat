(module
 (global i32 (i32.const 1) (i32.const 2)))