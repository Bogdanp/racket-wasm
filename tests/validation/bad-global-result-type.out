(#f . "result type error\n  expected: i32\n  found: i64\n  at index: 0\n  in: [i64]\n  at: (global (globaltype i32 #f) (vector (instr:i64.const #<loc> 1)))")